large-transfer-out-py/package.json

```diff
@@ -4,8 +4,32 @@
   "repository": "https://github.com/forta-network/starter-kits/tree/main/large-transfer-out-py",
   "description": "Bot identifies large native asset transfers that didn't exist X days ago",
   "chainIds": [
-    1, 56, 137, 43114, 42161, 10, 250
+    1,
+    56,
+    137,
+    43114,
+    42161,
+    10,
+    250
   ],
+  "chainSettings": {
```

Comment:
if we consider this table to see what should be the average processing time of a tx, and let's say the average process time of a tx in the bot is 400ms, then `shards` should be 5 for ethereum and 10 for bsc and polygon. so here I believe we can reduce ethereum shards to 2. wdyt?

        Chain: Blocktime, Number of Tx -> Avg processing time in ms target
        Ethereum: 12s, 150 -> 80ms
        BSC: 3s, 70 -> 43ms
        Polygon: 2s, 50 -> 40ms
        Avalanche: 2s, 5 -> 400ms
        Arbitrum: 1s, 5 -> 200ms
        Optimism: 24s, 150 -> 160ms
        Fantom: 1s, 5 -> 200ms

Reply:
Yea true.